controllers/onepassworditem_controller_test.go

```diff
@@ -105,7 +112,7 @@ var _ = Describe("OnePasswordItem controller", func() {
 				for k, v := range newData {
 					item.Fields = append(item.Fields, &onepassword.ItemField{Label: k, Value: v})
 				}
-				item.Version = item1.Version + 1
+				item.Version = randomInt()
```

Comment:
Usually in these scenarios the item version will increase. It would be weird to update the item with an older version of it. 

I wonder if there is a way to enforce some increase in here to simulate the real scenario.

Also, if let's say we have an item with version `120` and then we update it because it changed to version `15`, then we have a bug.

Reply:
I see your concern. You're right if the version is increased sequentially that's not an ideal approach.

So, the problem is that
- tests runs in parallel
- we use single k8sClient instance for all the test
- we use the same onepassword item `name+version` in several specs.

I see 2 more workarounds that might be better:
1. Create k8sClient instance for each spec (will be multiple instances, that 
2. Use unique onepassword item names in each spec.

I believe one of those 2 should cope with the issue.